pkgs/by-name/fl/flent/package.nix

```diff
@@ -56,11 +43,13 @@ python3Packages.buildPythonApplication rec {
     )
   '';
 
+  passthru.updateScript = nix-update-script { };
   meta = {
     description = "FLExible Network Tester";
     homepage = "https://flent.org";
     license = lib.licenses.gpl3;
     maintainers = with lib.maintainers; [ mmlb ];
     mainProgram = "flent";
+    platforms = lib.platforms.linux;
```

Comment:
Flent supports macOS according to https://pypi.org/project/flent . Why is this?

Reply:
This package has never been enabled for macos due to xvfb-run, now that I removed that the tests fail so I explicitly limited to linux for now and will try to figure the tests issue on macos layer.

Reply:
Would be better to mark `badPlatforms = lib.platforms.darwin` then.

Reply:
Thanks for the suggestion, updated.